docs/5-development/07-ts.md

```diff
@@ -71,19 +65,55 @@ class MyClass extends UI5Element {
 }
 ```
 
-### Defining the tag and other settings
+### Defining the tag, static getters and other settings
 
-Use `@customElement` as a replacement for the metadata `tag` setting.
+Use `@customElement` as a replacement for the metadata `tag` setting and all the `static get` methods.
 
-Use `@themeAware`, `@languageAware` and `@fastNavigation` where needed. Note that these do not accept parameters.
+Use `themeAware`, `languageAware` and `fastNavigation` where needed.
 
-Example:
+**Example**
+
+Instead of :
```

Comment:
Just one more thing, I suggest instead of making the comparison, just to describe the latest format